libs/testing/src/user.session.ts

```diff
@@ -63,6 +64,7 @@ const EMAIL_BLOCK: IEmailBlock[] = [
 ];
 
 export class UserSession {
+  public messageService = new MessageService();
```

Comment:
if this one is public maybe we should use this instance in the `events.ts` instead of putting it here?

Reply:
No, this meant to be private. Good spot.